java-backend/src/main/java/org/kframework/backend/java/symbolic/InitializeRewriter.java

```diff
@@ -192,7 +189,6 @@ public RewriterResult execute(K k, Optional<Integer> depth) {
             KOREtoBackendKIL converter = new KOREtoBackendKIL(module, definition, termContext.global(), false);
             ResolveSemanticCasts resolveCasts = new ResolveSemanticCasts(true);
             ExpandMacros macroExpander = new ExpandMacros(module, files, kompileOptions, false);
-            termContext.setKOREtoBackendKILConverter(converter);
```

Comment:
i cannot see why this is necessarily safe.

Reply:
because `setKOREtoBackendKILConverter` was deleted.